javascript/javascript2/week1/short.js

```diff
@@ -0,0 +1,16 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortestWord(array) {
+  let shortest = "";
+
+  for (let element of array) {
+    if (
+      typeof element === "string" &&
+      (shortest === "" || element.length < shortest.length)
+    ) {
+      shortest = element;
+    }
+  }
+  return shortest;
+}
+console.log("The shortest word is: " + shortestWord(danishWords));
```

Comment:
Good work!

What if there are shortest two words? Does this function return both or only one?